pyccel/ast/core.py

```diff
@@ -1162,46 +1195,90 @@ def __init__(
         imports = tuple(imports)
 
         self._name = name
-        self._variables = variables
+        self._variables = [v.lhs if isinstance(v, Assign) else v for v in variables]
+        self._variable_inits = [v.rhs if isinstance(v, Assign) else None for v in variables]
         self._funcs = funcs
+        self._init_func = init_func
+        self._free_func = free_func
+        self._program   = program
         self._interfaces = interfaces
         self._classes = classes
         self._imports = imports
         super().__init__()
 
     @property
     def name(self):
+        """ Name of the module
+        """
         return self._name
 
     @property
     def variables(self):
```

Comment:
can we say here module global variable instead of this desc?

Reply:
Fixed 42b1ebf5